hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/schemas/V052AddressBookSchema.java

```diff
@@ -59,6 +59,12 @@ public Set<StateDefinition> statesToCreate() {
     @Override
     public void migrate(@NonNull final MigrationContext ctx) {
         requireNonNull(ctx);
+
```

Comment:
Since we deleted the code for write config and pem, we could have deleted this code also. It shouldn't matter though. But that would have been consistent.